Sources/TokamakDemo/main.swift

```diff
@@ -26,6 +26,20 @@ struct CustomScene: Scene {
   }
 }
 
+struct CountDemo: View {
```

Comment:
Is this view still needed? I don't think it's currently used anywhere

Reply:
Used that to test the `onAppear` state update, I'll get it removed.